app/pages/project/instances/instance/tabs/StorageTab.tsx

```diff
@@ -128,7 +125,10 @@ export function StorageTab() {
       {
         label: 'Detach',
         disabled: !instanceCan.detachDisk(instance) && (
-          <>Instance must be in state {detachableStates} before disk can be detached</>
+          <>
+            Instance must be <span className="text-default">stopped</span> before disk can
+            be detached
```

Comment:
might as well do what the other ones, fudge it grammatically to save a few words: “instance must be stopped to detach disks” or maybe even “instance must be stopped” by itself? the button is disabled and it says “detach”